.github/workflows/ci.yml

```diff
@@ -12,6 +12,6 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: "18.x"
+          node-version: "18.0"
```

Comment:
Why do you fix it ?

Reply:
It's because `URL.canParse` doesn't exist in Node.JS v18.0, since it was only added in Node.JS v18.17.

So this test will make sure that we don't accidentally add anything that breaks with old versions of Node.JS v18. (if we do, we'd ideally want to make a new MAJOR change and state that the minimum supported of Node.JS has changed in the release notes).

Reply:
Oh, that's a clever move. Thanks for explanation.